MultimodalQnA/tests/test_compose_on_gaudi.sh

```diff
@@ -14,10 +14,10 @@ WORKPATH=$(dirname "$PWD")
 LOG_PATH="$WORKPATH/tests"
 ip_address=$(hostname -I | awk '{print $1}')
 
-export image_fn="apple.png"
+export image_fn="sample.png"
 export video_fn="WeAreGoingOnBullrun.mp4"
-export audio_fn="sample_audio.mp3"
-export caption_fn="apple.txt"
+export audio_fn="sample.mp3"
+export caption_fn="sample.txt"
```

Comment:
Are the naming conventions/requirements changing here, or is this just changing to improve consistency?

Reply:
I can add a comment in the code, but these all need to have the same base name in order to use audio as a caption